### PR TITLE
Client CSS: Beautifying

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Just a minor align fix

File: style/client.css [hunk 1]
```diff
@@ -15,10 +15,10 @@ body {
 
 	/* see http://pokemonshowdown.com/credits */
 	background: #344b6c url(../fx/client-bg-charizards.jpg) no-repeat left center fixed;
-  -webkit-background-size: cover;
-  -moz-background-size: cover;
-  -o-background-size: cover;
-  background-size: cover;
+  	-webkit-background-size: cover;
```
Comment (Zarel) [inline style/client.css]: should not have leading spaces :|

Comment (DeathlyPlays) [inline style/client.css]: Oops

**Comment by Zarel** [comment]
Looks good, thanks!

**Comment by DeathlyPlays** [comment]
No problem @Zarel my pleasure (seriously my pleasure it hurt my eyes)

**Pull request merged**